src/components/Login-Page/Login.jsx

```diff
@@ -0,0 +1,33 @@
+import { useContext } from 'react';
+import { UserContext } from '../../context/UserProvider';
+import { Link } from 'react-router-dom';
+import '../../styles/login.css'
+
+const Login = () => {
+    let {user} = useContext(UserContext)
+    if (!user) {
+        return <div>Loading...</div>;
+        }
+    user = user.data.user
+    return (
+        <main className='login'>
+            <h1 >Welcome {user.username}</h1>
+            <p>This is my first ever Portfolio which allows users to view all Articles and Comments on the articles. Articles can be deleted and updated, and new Articles can also be posted. Comments have similar functionality, allowing you to post comment(s) on various Articles and updating or deleting the Comments. Articles and Comments can be up-voted or down-voted.</p>
```

Comment:
This is nice and helps explain your project well. You could always put this on an /about page and link to it on your homepage but that's completely your choice.